webportal/src/app/cluster-view/services/services.component.js

```diff
@@ -66,3 +66,30 @@ $(document).ready(() => {
 });
 
 module.exports = { loadservices }
+
+// This function will call kubernetes restful api to get node - podlist - label info, to support service view monitor page.
+function getServiceView(kubernetesUrl) {
+  var nodesInfo = []
+  var request = require('sync-request');
```

Comment:
Did you include `sync-request` package in `package.json`?

Reply:
I will refine this later